move/walrus_site/sources/site.move

```diff
@@ -212,4 +243,44 @@ module walrus_site::site {
         } = site;
         id.delete();
     }
+
+    #[allow(lint(self_transfer))]
+    /// Define a Display for the Site objects.
+    public fun set_site_display(publisher: &Publisher, ctx: &mut TxContext) {
```

Comment:
I don't think this is the right way to do it. You'll end up creating Display objects for every update, and instead, one should have 1 display object with updates in this object.

Reply:
So you probably want to make this private and call it in the `init` instead.